src/index.ts

```diff
@@ -1,20 +1,21 @@
-import { createOffice, instances, ServiceType } from './createOffice';
-import * as Mail from './Mail';
-import { GmailAuthType } from './offices/GmailOffice';
-import { compileLodashFileTemplate } from './templates/compileLodashFileTemplate';
-import * as compileLodashTemplate from './templates/compileLodashTemplate';
+import { get } from 'lodash';
+import { instances } from './createOffice';
+import { Office } from './offices/Office';
+
+export * from './createOffice';
+export * from './Mail';
+export * from './offices/FcmOffice';
+export * from './offices/GmailOffice';
+export * from './offices/OnesignalOffice';
+export * from './offices/Office';
+export * from './offices/SendgridOffice';
+export * from './templates/compileLodashFileTemplate';
+export * from './templates/compileLodashTemplate';
+export * from './templates/withDefaultContext';
```

Comment:
How are globs better than the explicit exports? The index should only export explicitly IMO, so you cannot change the public API by accident.

Reply:
Replaced with explicit exports